include/openspace/scene/scene.h

```diff
@@ -248,7 +248,26 @@ class Scene : public properties::PropertyOwner {
      *
      * \return A list of all unique tags that are used in the currently loaded scene.
      */
-    std::vector<std::string> allTags();
+    std::vector<std::string> allTags() const;
+
+    /**
+     * Set a custom order for items in a given branch in the Scene GUI tree.
+     *
+     * \param guiPath The GUI path for which to set the order
```

Comment:
```suggestion
     * \param guiPath The GUI path for which to set the order.
```

Reply:
We need to decide on a style for this... Both for the Lua parameters and the regular doxygen comments. 

Should they end with a period, or not?

Currently, we actually seem to not use periods on either of them. But maybe we want to?  Would look better on the docs page, at least. E.g. [here](https://docs.openspaceproject.com/en/latest/generated/scripting-api/openspace.pathnavigation.html#createpath)

Reply:
If we decide to put periods, I could go through and change it everywhere. 

It should be apparent how it is done to anyone writing one of these descriptions, to avoid spending time on wondering. So it's important that we're consistent.

Reply:
I think I'd rather take my comment back for now, but I'll put it on the Dev Review discussion list.   Since most of the `param` descriptions are not actual sentences, I don't think there'd be need for a period at the end, but I don't feel strongly about it either